core/prelude/types.carbon

```diff
@@ -11,6 +11,7 @@ export import library "prelude/types/bool";
 // noise in the toolchain tests.
 // import library "prelude/types/i32";
 
+fn BigInt() -> type = "bigint.make_type";
```

Comment:
```suggestion
fn BigInt() -> type = "big_int.make_type";
```

Suggesting for the underscore between words.

Reply:
Done throughout.